packages/sdk-middleware-http/test/http.spec.js

```diff
@@ -206,7 +206,7 @@ describe('Http', () => {
         expect(res.request).toEqual(
           expect.objectContaining({
             headers: {
-              'Content-Type': ['application/json'],
+              'Content-Type': 'application/json',
```

Comment:
I couldn't recall why we were using an array of values for some headers...I think it makes sense to keep it simple and declare it as a single value

Reply:
Your argument makes sense to me too 👍